src/Mantasflowers.Services/Services/Email/SendgridConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+namespace Mantasflowers.Services.Services.Email
+{
+    public class SendgridConfiguration
+    {
+        public string ApiKey { get; set; }
+
+        public string From { get; set; }
```

Comment:
FromEmail or rather SenderEmail/ReceiverEmail might be better, since, at the moment, you'd have to look at code to be sure what this field is for.